doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2451 @@
+
+::
+
+  print(a[0])
+
+**Output:**
+
+::
+
+  [1 2 3 4]
+
+
+More information about arrays
+-----------------------------
+
+::
+
+  1D array, 2D array, ndarray, vector, matrix
+
+------
+
+You might occasionally hear an array referred to as a "ndarray," which is shorthand for "N-dimensional array." An N-dimensional array is simply an array with any number of dimensions. You might also hear **1-D**, or one-dimensional array, **2-D**, or two-dimensional array, and so on. The NumPy ``ndarray`` class is used to represent both matrices and vectors. A **vector** is an array with a single column, while a **matrix** refers to an array with multiple columns.
+
+**What are the attributes of an array?**
+
+An array is usually a fixed-size container of items of the same type and size. The number of dimensions and items in an array is defined by its shape. The shape of an array is a tuple of non-negative integers that specify the sizes of each dimension. 
+
+In NumPy, dimensions are called **axes**. This means that if you have a 2D array that looks like this:
+
+::
+
+  [[0., 0., 0.],
+   [1., 1., 1.]]
+
+Your array has 2 axes. The first axis has a length of 2 and the second axis has a length of 3.
+
+Just like in other Python container objects, the contents of an array can be accessed and modified by indexing or slicing the array. Different arrays can share the same data, so changes made on one array might be visible in another. 
+
+Array **attributes** reflect information intrinsic to the array itself. If you need to get, or even set, properties of an array without creating a new array, you can often access an array through its attributes. 
+
+`Read more about array attributes here <https://docs.scipy.org/doc/numpy/reference/arrays.ndarray.html>`_ and learn about `array objects here <https://docs.scipy.org/doc/numpy-1.17.0/reference/arrays.html>`_.
+
+
+How to create a basic array
+---------------------------
+
+
+::
+
+  np.array()
+  np.zeros() 
+  np.ones() 
+  np.empty() 
+  np.arange() 
+  np.linspace()
+  dtype
+
+-----
+
+To create a NumPy array, you can use the function ``np.array()``.
+
+All you need to do to create a simple array is pass a list to it. If you choose to, you can also specify the type of data in your list. `You can find more information about data types here <https://numpy.org/devdocs/user/quickstart.html#arrays-dtypes>`_.
+
+::
+
+    import numpy as np
+
+    a = np.array([1, 2, 3])
+
+You can visualize your array this way:
+
+.. image:: images/np_array.png
+
+Besides creating an array from a sequence of elements, you can easily create an array filled with 0s:
+
+**Input:**
+
+::
+
+  np.zeros(2)
+
+**Output:**
+
+::
+
+  array([0., 0.])
+
+Or an array filled with 1s:
+
+**Input:**
+
+::
+
+  np.ones(2)
+
+**Output:**
+
+::
+
+  array([1., 1.])
+  
+Or even an empty array! The function ``empty`` creates an array whose initial content is random and depends on the state of the memory. 
```

Comment:
Would be good to add something like: The reason to use ``empty`` over ``zeros`` or similar is speed - just make sure to fill every element afterwards!

Reply:
Great! Done!